tables/mdm/mdm.go

```diff
@@ -43,9 +46,18 @@ type profileStatus struct {
 }
 
 type depStatus struct {
-	DEPCapable bool
+	DEPCapable  bool `json:"dep_capable"`
```

Comment:
optional: Please consider adding comments giving context for what these struct fields represent.